src/ventus/wm/modes/expose.js

```diff
@@ -9,25 +9,34 @@ define([
 ], function(_, Promise) {
 	'use strict';
 
+	var exposeOnRightClick = true;
+
 	var ExposeMode = {
+		// Enable/disable expose on right-click (true=disable / false=enable)
+		setExposeOnRightClick: function (value) {
+			exposeOnRightClick = value;
+		},
+
 		// Launch when plugin is registered
 		register: function() {
 			var self = this;
 
 			console.log('Expose mode registered.');
 
-			this.el.on('contextmenu', _.throttle(function() {
-				// Right click sets expose mode
-				if (self.mode !== 'expose') {
-					if(self.windows.length > 0) {
-						self.mode = 'expose';
+			if (exposeOnRightClick === true) {
+				this.el.on('contextmenu', _.throttle(function() {
```

Comment:
If the goal of `setExposeOnRightClick` is to be able to dynamically enable/disable this option, the registration of the event listener should happen there.

However, to avoid register/unregister the listener every time the option changes, we could just check the value of exposeOnRightClick inside the listener code.